### PR TITLE
New actinia_core installer

### PR DESCRIPTION
**Pull request by neteler**
actinia_core (https://github.com/mundialis/actinia_core) is an OSGeo community project.
This PR proposes an installer script.

### CODE REVIEW

**Comment by kalxas** [comment]
Please fill in an application and submit to OSGeoLive mailing lists so we can pass a motion.

File: bin/install_actinia.sh [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+# get source code
+# https://github.com/mundialis/actinia_core
+cd "$TMP_DIR"
+git clone --single-branch https://github.com/mundialis/actinia_core.git
+cd actinia_core/
+
+# generate virtual env for Python3
+apt -y install virtualenv
+apt -y install python3-dev
+virtualenv -p python3 venv
```
Comment (kalxas) [inline bin/install_actinia.sh]: virtualenv, pip and conda are not permitted/used in OSGeoLive. For python software we need to have debian packages to install from.

**Comment by neteler** [comment]
The replacement of the `pip install -r requirements.txt` call may look like this:

```
apt -y install python3-boto3 python3-docutils python3-flask python3-flask-restful \
    python3-geopandas gunicorn python3-numpy python3-pandas python3-passlib python3-ply \
    python3-magic python3-shapely python3-sphinx redis python3-requests uwsgi
```

Here I am unsure if ok as no "python3-xxx" available:
```
# no py3? or ok?
apt -y install python-flask-httpauth python-scikits-learn
```

And here I am not aware of Ubuntu packages, how to install?  (I am more familiar with the RPM world...)
```
# flask-restful-swagger-2 flask-cors fluent-logger google-cloud rq
```

File: bin/install_actinia.sh (hunk 1 above)
Comment (neteler) [reply bin/install_actinia.sh]: The replacement of the `pip install -r requirements.txt` call may look like this:

```
apt -y install python3-boto3 python3-docutils python3-flask python3-flask-restful \
    python3-geopandas gunicorn python3-numpy python3-pandas python3-passlib python3-ply \
    python3-magic python3-shapely python3-sphinx redis python3-requests uwsgi
```

Here I am unsure if ok as no "python3-xxx" available:
```
# no py3? or ok?
apt -y install python-flask-httpauth python-scikits-learn
```

And here I am not aware of Ubuntu packages, how to install?  (I am more familiar with the RPM world...)
```
# flask-restful-swagger-2 flask-cors fluent-logger google-cloud rq
```

Comment (kalxas) [reply bin/install_actinia.sh]: ok we can create the missing dependencies, no worries.

**Comment by neteler** [comment]
> Please fill in an application and submit to OSGeoLive mailing lists so we can pass a motion.

For the record: https://lists.osgeo.org/pipermail/osgeolive/2019-July/013931.html

File: bin/install_actinia.sh (hunk 1 above)
Comment (neteler) [reply bin/install_actinia.sh]: To avoid the virtualenv, we could hardcode "python3" in this script.

**Comment by kalxas** [comment]
Thanks @neteler

File: bin/install_actinia.sh (hunk 1 above)
Comment (kalxas) [reply bin/install_actinia.sh]: it is not only virtualenv, we avoid pip too.

**Comment by darkblue-b** [comment]
@neteler  only you know your target audience best;  with some python packages, the cost of the python wrappers is small, due to linked libs.  Python 2.7 is not illegal on **osgeolive** !

File: bin/install_actinia.sh (hunk 1 above)
Comment (kalxas) [reply bin/install_actinia.sh]: actinia and dependencies now have a debian package:

https://launchpad.net/~gcpp-kalxas/+archive/ubuntu/osgeolive/+packages

Comment (kalxas) [reply bin/install_actinia.sh]: Actually the package python3-actinia-core will pull this list of dependencies automatically through apt.

**Pull request merged**